nim_status/accounts/accounts.nim

```diff
@@ -74,6 +74,23 @@ proc getAccounts*(db: DbConn): seq[Account] =
                     ORDER BY  {tblAccounts.createdAt.columnName} ASC"""
   result = db.all(Account, query)
 
+proc getChatAccount*(db: DbConn): Account =
+  var tblAccounts: Account
+  let query = fmt"""SELECT    {tblAccounts.address.columnName},
+                              {tblAccounts.wallet.columnName},
+                              {tblAccounts.chat.columnName},
+                              {tblAccounts.`type`.columnName},
+                              {tblAccounts.storage.columnName},
+                              {tblAccounts.path.columnName},
+                              {tblAccounts.publicKey.columnName},
+                              {tblAccounts.name.columnName},
+                              {tblAccounts.color.columnName},
+                              {tblAccounts.createdAt.columnName},
+                              {tblAccounts.updatedAt.columnName}
+                    FROM      {tblAccounts.tableName}
+                    WHERE     {tblAccounts.chat.columnName} = TRUE"""
```

Comment:
We are using `0`/`1` instead of `FALSE`/`TRUE` in `getWalletAccounts` (and a couple others in #240). I have no strong preference either way, but maybe we should keep it consistent?

Reply:
SQLite (and so SQLCipher) auto-translates `FALSE / TRUE` in queries to `0 / 1` and on the nim side we have the field as `Option[bool]`, so I think it's more natural to use `FALSE / TRUE`. At the same time, consistency is important, so we should decide which way we want to do it and then change all the queries to use the same style.